code/modules/mod/mod_construction.dm

```diff
@@ -0,0 +1,207 @@
+/obj/item/mod/construction
+	desc = "A part used in MOD construction."
+
+/obj/item/mod/construction/helmet
+	name = "MOD helmet"
+	icon_state = "helmet"
+
+/obj/item/mod/construction/chestplate
+	name = "MOD chestplate"
+	icon_state = "chestplate"
+
+/obj/item/mod/construction/gauntlets
+	name = "MOD gauntlets"
+	icon_state = "gauntlets"
+
+/obj/item/mod/construction/boots
+	name = "MOD boots"
+	icon_state = "boots"
+
+/obj/item/mod/construction/core
+	name = "MOD core"
+	icon_state = "mod-core"
+	desc = "A mystical crystal able to convert cell power into energy usable by MODsuits."
+
+/obj/item/mod/construction/armor
+	name = "MOD standard armor plates"
+	desc = "Armor plates used to finish a MOD"
+	icon_state = "armor"
+	var/theme = /datum/mod_theme
+
+/obj/item/mod/construction/armor/engineering
+	name = "MOD engineering armor plates"
+	icon_state = "engineering-armor"
+	theme = /datum/mod_theme/engineering
+
+/obj/item/mod/paint
+	name = "MOD paint kit"
+	desc = "This kit will repaint your MODsuit to something unique."
+	icon = 'icons/obj/mod.dmi'
+	icon_state = "paintkit"
+
+/obj/item/mod/construction/shell
+	name = "MOD shell"
+	icon_state = "mod-construction"
+	desc = "An empty MOD shell."
+	var/obj/item/core
+	var/screwed_core = FALSE
+	var/obj/item/helmet
+	var/obj/item/chestplate
+	var/obj/item/gauntlets
+	var/obj/item/boots
+	var/wrenched_assembly = FALSE
+	var/screwed_assembly = FALSE
+	var/icon_to_use
+
+/obj/item/mod/construction/shell/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	if(!core && istype(I, /obj/item/mod/construction/core)) //Construct
+		if(!user.transferItemToLoc(I, src))
+			return
+		playsound(src, 'sound/machines/click.ogg', 30, TRUE)
+		to_chat(user, span_notice("You insert [I] into [src]."))
+		core = I
+		icon_to_use = "core"
+
+	if(core)
+		if(I.tool_behaviour == TOOL_SCREWDRIVER) //Construct
+			if(I.use_tool(src, user, 0, volume=30))
+				to_chat(user, span_notice("You screw [I] into [src]."))
```

Comment:
"you screw screwdriver into src"

what?

anyways, this balloon alert should just be "secured [I]"